test/read.spec.ts

```diff
@@ -65,4 +65,25 @@ describe("read .nodesecurerc", () => {
       strategy: "none"
     });
   });
+
+  it("should return the default value 'null' when the memoize option is not declared ", async() => {
+    await read(location, { createIfDoesNotExist: true });
+    expect(memoized()).to.eq(null);
+  });
+
+  it("should return the configuration passed in parameter", async() => {
+    await read(location, { createIfDoesNotExist: true, memoize: true });
+    expect(memoized()).to.deep.equal(generateDefaultRC());
+  });
+
+  it("should merge with the previous stored payload", async() => {
+    const payload: Partial<RC> = {
+      version: "2.0.0",
+      i18n: "french",
+      strategy: "snyk"
+    };
+    memoize(payload, { overwrite: false });
+
+    expect(memoized()).to.deep.equal({ ...generateDefaultRC, ...payload });
+  });
```

Comment:
Not sure what's the goal of this test here?

Reply:
The way I saw memoization, it's between two calls of the function `read`. Did I miss something?

Reply:
In the `read` method, we use the `memoize` API with the `overwrite` option set to false. To ensure that this method correctly merges the current configuration with the previously stored one, I performed this test.

Reply:
There no read call in the test, it doesn't test anything related to read...

Reply:
I did not make a call since the previous test does. Since both tests are in the same scope, I assumed that the value would still be cached.

Reply:
No you can't mix unit tests between them (and anyway you have no guarantee on the state).